returnn/datasets/postprocessing.py

```diff
@@ -15,7 +15,7 @@
 from .basic import init_dataset
 from .cached2 import CachedDataset2
 
-__all__ = ["PostprocessingDataset"]
+__all__ = ["PostprocessingDataset", "LaplaceOrdering", "T", "Sequential"]
```

Comment:
The `T` does not need to be exported.
```suggestion
__all__ = ["PostprocessingDataset", "LaplaceOrdering", "Sequential"]
```